### PR TITLE
refactor: all plugin use helm refactor

### PR DESCRIPTION
**Pull request by steinliber**
Signed-off-by: Meng JiaFeng <jiafeng.meng@merico.dev>

## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.
- [x] My code has the necessary comments and documentation (if needed).
- [ ] I have added relevant tests

## Description
- refactor all plugins which use helm to install
- all plugin has been tested

## Related Issues
Subtask of #850

### CODE REVIEW

File: internal/pkg/plugin/harbor/harbor.go [hunk 1]
```diff
@@ -1 +1,9 @@
 package harbor
+
+var defaultDeploymentList = []string{
+	"habor-chartmuseum",
```
Comment (iyear) [inline internal/pkg/plugin/harbor/harbor.go:4]: typo

File: internal/pkg/plugin/harbor/harbor.go [hunk 2]
```diff
@@ -1 +1,9 @@
...
+	"harbor-core",
+	"harbor-jobserice",
```
Comment (iyear) [inline internal/pkg/plugin/harbor/harbor.go:6]: It's typo?

Comment (iyear) [reply internal/pkg/plugin/harbor/harbor.go:6]: image

Comment (iyear) [reply internal/pkg/plugin/harbor/harbor.go:6]: Maybe they are `harbor-chartmuseum` and `harbor-jobservice`

**Review by IronCore864 — APPROVED** [review]

**Review by daniel-hutao — APPROVED** [review]

**Pull request merged**